packages/eslint-config-sheriff/package.json

```diff
@@ -2,15 +2,21 @@
   "name": "eslint-config-sheriff",
   "version": "18.7.0",
   "description": "A comprehensive and opinionated Typescript-first ESLint configuration.",
-  "main": "./dist/index.js",
+  "main": "./dist/index.cjs",
```

Comment:
i thunk we can probably just removing this

Reply:
Yeah. I think exports overrides it, but I didn't want to remove it if it was there intentionally.

Reply:
Ya so, `exports` overrides it, but i think it is used for some backward compatibility.
I don't really mind either way, i thought `"main": "./dist/index.js"` was fine, not sure what `publint` says in this case.